static/app/components/alert.tsx

```diff
@@ -13,17 +13,12 @@ type Props = {
   system?: boolean;
   expand?: React.ReactNode[];
   expandIcon?: React.ReactNode;
+  opaque?: boolean;
```

Comment:
When would we ever want it to actually not be opaque? haha

Reply:
imo, most of the time 😛 I prefer semi-transparent backgrounds cause they let the color from behind bleed through, resulting in alerts that blend in more with the UI surrounding them.

Here's a comparison - the first is opaque, the second is semi-transaprent:
<img width="791" alt="Screen Shot 2022-01-03 at 4 09 37 PM" src="https://user-images.githubusercontent.com/44172267/147993511-b3aad551-8bd6-4dce-b473-348a550689dc.png">

Reply:
👍